chart/k8gb/templates/external-dns/external-dns.yaml

```diff
@@ -41,4 +41,18 @@ spec:
             cpu: "500m"
         securityContext:
           readOnlyRootFilesystem: true
+        {{- if .Values.rfc2136.rfc2136auth.gssTsig.enabled }}
+        volumeMounts:
+          - mountPath: /etc/krb5.conf
+            name: kerberos-config-volume
+            subPath: krb5.conf
+        {{- end }}
+      {{- if .Values.rfc2136.rfc2136auth.gssTsig.enabled }}
```

Comment:
can't this be simplified to one block? The condition is the same, is there an issue with indentation?